ros2action/ros2action/verb/echo.py

```diff
@@ -19,6 +19,7 @@
 import threading
 
 import rclpy
+
```

Comment:
This blank line is better to be removed.
Module and submodule are usually grouped together.